src/index.html

```diff
@@ -11,7 +11,126 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-group">
+        <legend class="name">Personal information</legend>
+        <label class="form-field">Surname:
```

Comment:
Text for label should be on the new line, check and fix in all places
```suggestion
        <label class="form-field">
          Surname:
```